tests/acceptance/manager/ModelGroupsActionsCest.php

```diff
@@ -0,0 +1,148 @@
+        $I->waitForPageUpdate();
+        $I->waitForText('Name cannot be blank.');
+        $I->click("//button[contains(@class, 'add-item')]");
+        $I->click("//button[contains(@class, 'add-item')]");
+        foreach (range(0,2) as $i) {
+            (new Input($I, "//input[@name='ModelGroup[$i][name]']"))
+                ->setValue("TEST_" . $this->uni . "_New_" . $i);
+            (new Input($I, "//textarea[contains(@name, 'ModelGroup[$i][descr]')]"))
+                ->setValue("Test description for $i item");
+            (new Input($I, "//input[@name='ModelGroup[$i][limit_dtg]']"))
+                ->setValue(($i + 1) * 10);
+            (new Input($I, "//input[@name='ModelGroup[$i][limit_sdg]']"))
+                ->setValue(($i + 1) * 10);
+            (new Input($I, "//input[@name='ModelGroup[$i][limit_m3]']"))
+                ->setValue(($i + 1) * 10);
+            (new Input($I, "//input[@name='ModelGroup[$i][limit_twr]']"))
+                ->setValue(($i + 1) * 10);
+        }
+        $I->pressButton('Save');
+        $I->waitForPageUpdate();
+        $I->closeNotification('Created');
+        $I->seeInCurrentUrl('/stock/model-group/index');
+
+        $name = 'TEST_' . $this->uni . '_';
+        $selector = "//input[contains(@name, 'ModelGroupSearch[name_ilike]')]";
+        $this->index->filterBy(new Input($I, $selector), $name);
+        $count = $this->index->countRowsInTableBody();
+        foreach (range(1, $count) as $i) {
+            $I->see(($i) * 10, "//tbody/tr[$i]");
+        }
+    }
+
+    /**
+     * @param Manager $I
+     * @throws \Codeception\Exception\ModuleException
+     */
+    public function ensureCopyModelGroupWorks(Manager $I): void
+    {
+        $count = $this->startAction($I, 'New');
+        $I->pressButton('Copy');
+        $this->fillInput($I, $count, 'Copy');
+        $I->closeNotification('Created');
+        $I->seeInCurrentUrl('/stock/model-group/index');
+    }
+
+    /**
+     * @param Manager $I
+     * @throws \Codeception\Exception\ModuleException
+     */
+    public function ensureUpdateModelGroupWorks(Manager $I): void
+    {
+        $count = $this->startAction($I, 'Copy');
+        $I->pressButton('Update');
+        $this->fillInput($I, $count, 'Updated');
+        $I->closeNotification('Updated');
+        $I->seeInCurrentUrl('/stock/model-group/index');
+    }
+
+
+    /**
+     * @param Manager $I
+     * @throws \Codeception\Exception\ModuleException
+     */
+    public function ensureDeleteModelGroupWorks(Manager $I): void
+    {
+        $this->startAction($I, '');
+        $I->pressButton('Delete');
+        $I->acceptPopup();
+        $I->closeNotification('Deleted');
+        $I->see('No results found.', '//tbody');
+    }
+
+    /**
+     * @param Manager $I
+     * @param string $lastOperations
+     * @return int
+     * @throws \Codeception\Exception\ModuleException
+     */
+    private function startAction(Manager $I, string $lastOperations): int
```

Comment:
```suggestion
    private function filterModelsByNameAndSelectThem(Manager $I, string $lastOperations): int
```

На самом деле метод никакого действия не начинает. Он фильтрует группы моделей по именам и выбирает их в таблице. Называй метод соответсвенно тому, что он на самом деле делает.